build/travis-debug/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Simulate our Maven build on Travis-CI
+# WORKDIR ends up as /home/travis/build
+# Source directory should be mounted here
+
+FROM quay.io/travisci/travis-jvm
```

Comment:
I don't know of any way to verify.  We could specify an [explicit tag](https://quay.io/repository/travisci/travis-jvm?tab=tags).  Looking at `docker history`, I don't see a referenced image.

We could re-build this against a standard image like the [`maven` images](https://hub.docker.com/_/maven/).